src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,42 @@
+@import "../../patternfly-utilities";
+
+$pf-l-gallery__item--Width: 250px !default;
+
+$pf-l-gallery--GridGap: 1rem !default;
+$pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax($pf-l-gallery__item--Width, 1fr)) !default;
+$pf-l-gallery__item--Flex: 0 1 $pf-l-gallery__item--Width !default;
+
 .pf-l-gallery {
-  display: grid;
-  grid-gap: 1rem; //revisit this value
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
-  padding: 0;
-  margin: 0;
+
+  display: flex;
+  flex-flow: row wrap;
+
+  &__item {
+    flex: $pf-l-gallery__item--Flex;
+  }
+
+  &.pf-has-gutter > *:not(:last-child) {
+    margin-right: $pf-l-gallery--GridGap;
+  }
+  &.pf-has-gutter > * {
+    margin-bottom: $pf-l-gallery--GridGap;
+  }
+
+
+  @supports (display: grid) {
+    display: grid;
+    grid-template-columns: $pf-l-gallery--GridTemplateColumns;
+
+    &.pf-has-gutter {
+      grid-gap: $pf-l-gallery--GridGap;
+    }
+    &.pf-has-gutter > *:not(:last-child) {
```

Comment:
if these 2 lines are negating something from the fallback please add a comment.

Reply:
Done